includes/sanitizers/class-amp-img-sanitizer.php

```diff
@@ -81,6 +81,7 @@ public function get_selector_conversion_mapping() {
 			Tag::IMG => [
 				'amp-img',
 				'amp-anim',
+				'amp-pixel',
```

Comment:
This is a thoughtful addition, but in practice a Facebook pixel should really _never_ be styled by authors. It's supposed to be invisible after all. By adding `amp-pixel` to the list here, it means that if there is a `img { … }` it will get sanitized into `amp-img, amp-anim, amp-pixel { … }` .

```suggestion
```